packages/shared/components/atoms/MediaDisplay.svelte

```diff
@@ -47,7 +49,7 @@
             case ParentMimeType.Video:
                 return 'video'
             default:
-                onError()
+                onWarning(localize('error.nft.unsupportedFileType.'))
```

Comment:
Calling the `localize` function here before the `short` or `long` suffix has been appended. I think you meant to just pass that `error.nft.unsupportedFileType` string into the `onWarning` function without the `localize` function call.

Reply:
true, and i didnt see any issue while coding it because 
`localize('error.nft.unsupportedFileType.')`
will again return `'error.nft.unsupportedFileType.'` which will later on be translated with `short/long`